app/views/templates/inputs/SingleCheckbox.scala.html

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  *@
 
+@this()
```

Comment:
Could you also please make this change (both `this()` and the capitalisation of the file) for every scala.html file in the service? The bug explicitly said "templates" so I can see why you focused on this directory, but it meant every HTML view/template. I raised the bug so apologies for not being descriptive enough.

The change means all these files become injectable classes that can manage their own dependencies instead of objects, which is more in line with most of our other types of files. So basically it's something we want for every twirl template. It should have been done in the initial Play 2.6 upgrade of this service but was missed.